packages/loader/container-loader/src/containerContext.ts

```diff
@@ -107,8 +107,9 @@ export class ContainerContext implements IContainerContext {
         return this.container.clientDetails;
     }
 
+    private _connected: boolean;
     public get connected(): boolean {
```

Comment:
Add a doc comment here explaining what this means?  You don't necessarily have to admit that the Loader may lie to it. (but you could). I'm thinking something like that when true, ops are free to flow, but when false, ops should be kept as pending, or whatever you want to say.